practice-two/src/javascript/modules/filterNotes.js

```diff
@@ -0,0 +1,42 @@
+import httpUtils from '../utils/httpUtils';
+import { renderNote, renderAllNotes } from './getNotes'; 
```

Comment:
@quanghuy1799 Why `renderAllNotes` from importing is same with your function defined and the end of this file?